src/main.cpp

```diff
@@ -78,8 +82,27 @@ string gLogFileName;
 /*******************************************************************************
  *
  ******************************************************************************/
+#ifdef WITH_SYSTEMD
+#define NOTIFY_SYSTEMD(...)														\
```

Comment:
... and only there you are really do something with systemd

Reply:
done